test/tests/happy.jsx

```diff
@@ -440,6 +440,34 @@ describe('zoid happy cases', () => {
         });
     });
 
+    it('should render a component into a nested shadow dom', () => {
+        return wrapPromise(({ expect }) => {
+
+            window.__component__ = () => {
+                return window.zoid.create({
+                    tag:    'test-render-nested-shadow-dom',
+                    url:    'mock://www.child.com/base/test/windows/child/index.htm',
+                    domain: 'mock://www.child.com'
+                });
+            };
+
+            onWindowOpen().then(expect('onWindowOpen', ({ win }) => {
+                if (getParent(win) !== window) {
+                    throw new Error(`Expected window parent to be current window`);
+                }
+            }));
+
+            const { container, destroy } = getContainer({ shadow: true, nested: true });
```

Comment:
@gabrielo91 and I were pairing on this and noticed that `slot: true` is missing. Adding it here helps the new nested use case to get hit. 

```suggestion
            const { container, destroy } = getContainer({ shadow: true, nested: true, slot: true });
```